src/stan/variational/families/normal_fullrank.hpp

```diff
@@ -49,12 +49,15 @@ namespace stan {
 
     public:
       // Constructors
+
+      // Constructs a zero-filled variational distribution
       explicit normal_fullrank(size_t dimension) :
         dimension_(dimension) {
         mu_     = Eigen::VectorXd::Zero(dimension_);
-        L_chol_  = Eigen::MatrixXd::Identity(dimension_, dimension_);
+        L_chol_ = Eigen::MatrixXd::Zero(dimension_, dimension_);
```

Comment:
I think it'd be cleaner with a k index on the outside and i and j on the inside rather than i, ii, and jj in calc_grad.  No big deal if you're already fed up with these comments.  We tend to use ii and jj for random effects indexing following Gelman and Hill's book.